backend/medtagger/database/models.py

```diff
@@ -62,6 +63,19 @@ def role(self) -> Role:
         return self.roles[0]
 
 
+class UserSettings(Base):
+    """Settings of user."""
+
+    __tablename__ = 'UserSettings'
+    id: int = Column(Integer, autoincrement=True, primary_key=True)
```

Comment:
Do not use separate values for `id` and `user_id`. This should be one field. Settings for given user should be stored under the same ID as owner's ID.

Look at this example: https://github.com/jpowie01/MedTagger/blob/backend_for_surveys/backend/medtagger/database/models.py#L368